pom.xml

```
@@ -27,7 +27,7 @@
     <BUILD_TIMESTAMP>${maven.build.timestamp}</BUILD_TIMESTAMP>
     <EMISSARY_VERSION>${project.version}</EMISSARY_VERSION>
     <argLine />
-    <checkstyleFormatter>${project.basedir}/contrib/checkstyle.xml</checkstyleFormatter>
+    <checkstyleFormatter>${project.basedir}/.github/linters/checkstyle.xml</checkstyleFormatter>
```

```suggestion
    <checkstyleFormatter>${project.basedir}/contrib/checkstyle.xml</checkstyleFormatter>
```
If you use relative paths or symlinks, this can go back to the original.